src/components/DifficultyExplanation/DifficultyExplanation.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import "./DifficultyExplanation.css";
+import { useState } from 'react';
+
+const DisplayInfo = ({difficulty}) => {
+    return (
+        <div className='displayInfo'>
+            {difficulty === "Beginner" && 
+                <div>
+                    <p>- Know nothing about the topic</p>
+                    <p>- Looking for a reintroduction from scratch</p>
+                    <p>- Source can be understood without prior knowledge"</p>
+                </div>
+            }
+            {difficulty === "Intermediate" && 
+                <div>
+                    <p>- Most developers know this stuff</p>
+                    <p>- Stuff to look into after you understand how the basics of a technology work</p>
+                    <p>- Helpful to better understand and work comfortably with a specific technology</p>
+                </div>
+            }
+            {difficulty === "Advanced" && 
+                <div>
+                    <p>- Extra skills</p>
+                    <p>- Things that are nice to know, but not needed, to be comfortable with or use the technology effectively</p>
+                    <p>- Ideas that are complicated to understand them unless you've worked with the technology for a while</p>
+                </div>
+            }
+        </div>    
+    )
+}
```

Comment:
We can turn this into a map and display the value of the map (and throw it into `resources.js` if you want)! EG
```
const difficultyTextMap = {
Beginner: ["- Know nothing ...", "- Looking for ...", "..."],
...
}
```

and then in the `JSX`

```
<>
{difficultyTextMap[difficulty].map((val) -> <p>{val}</p>)}
</>
```

I personally did not test this out so try it yourself :))

Reply:
Just for your info, if we were to keep this as is, we should put this underneath `DifficultyExplanation`. Generally code is read in a top down fashion like a book, so it would be great to put more general things on top

Reply:
ditto this!